main.nf

```diff
@@ -646,6 +647,67 @@ def d4_coverage_version(task) {
 	"""
 }
 
+process verifybamid2 {
+        cpus 16
+        memory '10 GB'
+        // publishDir "${OUTDIR}/contamination", mode: 'copy', overwrite: 'true', pattern: '*.selfSM'
+        tag "$id"
+        container = "${params.container_verifybamid2}"
+
+        input:
+                set group, id, file(bam), file(bai) from verifybamid2_bam
+
+        output:
+                file("${id}.result.selfSM")
+                file("${id}.result.Ancestry")
+                set group, file("*versions.yml") into ch_verifybamid2_versions
+
+        script:
+
+                if ( params.antype == "wgs") {
+                         """
+                         verifybamid2 \
+                         --SVDPrefix ${params.svdprefix} \
+                         --Reference $genome_file \
```

Comment:
What about `${genome_file}`, to keep things consistent

Reply:
Done!